subprojects/gusb.wrap

```diff
@@ -1,4 +1,4 @@
 [wrap-git]
 directory = gusb
 url = https://github.com/hughsie/libgusb.git
-revision = 43b327a1e213aff00833842c455a796a068077cd
+revision = 1281d6ff0789ea8e4a825d0c379d175254e269ee
```

Comment:
can we use a tag name here? Like `0.3.4`

Reply:
Done. Set to 0.3.3.